app/views/layouts/_analytics_body.html.slim

```diff
@@ -0,0 +1,9 @@
+- if track_analytics?
+  /! Google Tag Manager (noscript)
+  noscript
+    iframe(
+      src="https://www.googletagmanager.com/ns.html?id=#{ENV.fetch('TRACKING_ID')}"
```

Comment:
Accompanying the new `ApplicationController` predicate with a method that returned the tracking key would be best. Choosing names that show they are related is nice too.